### PR TITLE
Try generate compact entity files

### PR DESCRIPTION
**Pull request by billy1624**
Not intended to be merged

### CODE REVIEW

File: tests/common/bakery_chain/metadata.rs [hunk 1]
```diff
@@ -1,10 +1,11 @@
+//! SeaORM Entity. Generated by sea-orm-codegen 0.2.0
+
 use sea_orm::entity::prelude::*;
-use uuid::Uuid;
 
 #[derive(Clone, Debug, PartialEq, DeriveEntityModel)]
 #[sea_orm(table_name = "metadata")]
 pub struct Model {
-    #[sea_orm(primary_key, auto_increment = false)]
+    #[sea_orm(primary_key)]
```
Comment (tyt2y3) [inline tests/common/bakery_chain/metadata.rs:8]: We should add `auto_increment = false`

File: tests/common/bakery_chain/lineitem.rs [hunk 2]
```diff
@@ -1,11 +1,12 @@
+//! SeaORM Entity. Generated by sea-orm-codegen 0.2.0
+
 use sea_orm::entity::prelude::*;
 
 #[derive(Clone, Debug, PartialEq, DeriveEntityModel)]
 #[sea_orm(table_name = "lineitem")]
 pub struct Model {
     #[sea_orm(primary_key)]
     pub id: i32,
-    #[sea_orm(column_type = "Decimal(Some((19, 4)))")]
```
Comment (tyt2y3) [inline tests/common/bakery_chain/lineitem.rs:8]: Is it possible to get the required info from SeaSchema for this?

Comment (billy1624) [reply tests/common/bakery_chain/lineitem.rs]: Fixed on 39c5a4d. But I'm thinking what `column_type` should also be generated by codegen, perhaps `Text`?

File: tests/common/bakery_chain/metadata.rs (hunk 1 above)
Comment (billy1624) [reply tests/common/bakery_chain/metadata.rs]: Fixed on 39c5a4d as well

**Pull request closed without merging**